{"errorMessage": "Syntax error in module 'lambda_function': unexpected character after line continuation character (lambda_function.py, line 1173)", "errorType": "Runtime.UserCodeSyntaxError", "requestId": "", "stackTrace": ["  File \"/var/task/lambda_function.py\" Line 1173\n        return normalized\\n\\ndef _filter_lh_group_offers(offers: List[Dict[str, Any]]) -> List[Dict[str, Any]]:\n"]}